not_nil.go

```diff
@@ -4,29 +4,27 @@
 
 package validation
 
-import "errors"
-
 // NotNil is a validation rule that checks if a value is not nil.
 // NotNil only handles types including interface, pointer, slice, and map.
 // All other types are considered valid.
-var NotNil = notNilRule{message: "is required"}
+var NotNil = notNilRule{err: NewError("validation_not_nil_invalid", "is required")}
```

Comment:
`validation_not_nil_required`

Reply:
How about something like `is_blank` or `required`?

Reply:
We already have `validation_required`.  `is_blank` is not very clear.